src/artifact.ts

```diff
@@ -192,8 +192,9 @@ export type GreenwaveRequirementType = {
     subject_type: string;
     /* python2-flask-1.0.2-1.rawhide */
     subject_identifier: string;
-    result_id: number;
-    error_reason: string;
+    result_id?: number;
```

Comment:
@mgrabovsky do you have an example when there this entry in undefied?
Do you have buid NVR?

Reply:
See pcm-202211-0.el9, it's linked in the ticket mentioned above.

Reply:
Ok